agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -41,24 +40,14 @@ import (
 	"github.com/percona/pmm-agent/utils/version"
 )
 
-const defaultWaitTime = 60 * time.Second
-
 // PGStatMonitorQAN QAN services connects to PostgreSQL and extracts stats.
 type PGStatMonitorQAN struct {
-	q            *reform.Querier
-	dbCloser     io.Closer
-	agentID      string
-	l            *logrus.Entry
-	changes      chan agents.Change
-	monitorCache *statMonitorCache
-
-	// By default, query shows the actual parameter instead of the placeholder.
```

Comment:
Since pgsmNormalizedQuery and waitTime can be changed without restart its not good idea to cache it anymore. Now we checking settings everytime when we creating new bucket.

Reply:
👍🏼